### DISCUSSION TITLE
Propagate api results to next test

### DISCUSSION THREAD

**Discussion by MarcoFierimonte** [apideck-libraries/portman#397] [Q&A] [answered]
Hi all :) 

Is it possible, using some portman configuration, propagate an api test result and use it in the next operation? 
For example:
1. First test perform a GET that retrive from a Json response a "name" variable value.
2. Second test perform a GET using the "name" value from step 1.

Thanks for help.
Marco F.

**Comment by thim81** [comment, marked as answer]
Hi @MarcoFierimonte 

Portman uses [assignVariables](https://github.com/apideck-libraries/portman#portman---assignvariables-properties) to store response values and use them in subsequent requests or tests.

Have a look at the example: https://github.com/apideck-libraries/portman/tree/main/examples/testsuite-assign-variables

**Reply by MarcoFierimonte** [reply]
Thanks a lot @thim81 !
